lib/widgets/bento_box.dart

```diff
@@ -110,6 +111,43 @@ class BentoBox extends StatefulWidget {
     (children ?? []).forEach((c) => print('$c, <<>>'));
   }
 
+  static calculateCustomizedLayout(
```

Comment:
Do not add this to bento_box. Put it in your own jumbled_word_game file and pass it as function to layout